hcl2template/types.packer_config.go

```diff
@@ -244,6 +248,23 @@ func (cfg *PackerConfig) getCoreBuildPostProcessors(source SourceBlock, blocks [
 		if ppb.OnlyExcept.Skip(source.Type + "." + source.Name) {
 			continue
 		}
+
+		name := ppb.PName
+		if name == "" {
+			name = ppb.PType
+		}
+		// -except
+		exclude := false
+		for _, exceptGlob := range cfg.except {
+			if exceptGlob.Match(name) {
+				exclude = true
+				break
+			}
+		}
+		if exclude {
+			continue
```

Comment:
I think this needs to be a break per the [documentation](https://www.packer.io/docs/commands/build/#except-foo-bar-baz) :
```suggestion
			break
```